### PR TITLE
Cu 86bwkftck/be swagger/database/model/set up

### PR DESCRIPTION
**Pull request by Jidekas**
**Description of PR**

This PR
 - changes the branch name to the current convention agreed by the team
 - Eliminates unnecessary dependencies like cross-env and lodash-merge 
 - Maintains the ability to connect to local and cloud dbs dynamically.
 - env files will be named .env.development and .env.production
 - Eliminates compiling to js and runs ts using ts-node
 - Removes unnecessary comments and variable in various models

---

Please check all that apply.

#### Checklist:

- [x] Rebased with master
- [x] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---
**Screenshots**  
_add relevant screenshots here_

---
**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Comment by KristofaJosh** [comment]
Task linked: [CU-86bwkftck BE - Swagger/Database/Model/ set-up](https://app.clickup.com/t/86bwkftck)

File: server/src/models/groups.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    created_at: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    // profilePicture: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    // },
+  },
```
Comment (KristofaJosh) [inline server/src/models/groups.ts:138]: do we need this?

File: server/src/models/groups.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  amount_withdrawn: number
+  members: Members[]
+  slots: number[]
+  availableSlots: number[]
+  number_of_participants: number //number of participants
+  frequency: string //ft
+  duration: string
+  startDate: Date //start date
+  endDate: Date //end date
+  created_at: Date
```
Comment (KristofaJosh) [inline server/src/models/groups.ts:46]: some of the comments here aren't necessary

**Review by KristofaJosh — APPROVED** [review]

**Review by polymathmode — APPROVED** [review]

**Comment by KristofaJosh** [comment]
LGTM!

**Pull request merged**